src/core/translations/dutch-translations.constant.js

```diff
@@ -553,9 +553,30 @@ angular.module('udb.core')
+          'Cursus met open sessies': '<p>Geef een enthousiaste omschrijving van de workshop of cursus met open sessies. </p> <ul> <li>Wat leer je er precies? </li> <li>Is er <strong>voorkennis</strong> nodig om deel te nemen? </li> <li>Wat is het <strong>niveau</strong> van de lessen? </li> <li>Krijg je op het einde een <strong>diploma</strong> of certificaat?</li> </ul>',
+          'Kamp of vakantie': '<p>Geef een enthousiaste omschrijving van het kamp of de vakantie. </p> <ul> <li>Welke <strong>activiteiten</strong> kan je er doen? </li> <li>Is er een <strong>thema</strong>? </li> <li>Is het kamp <strong>met of zonder overnachtingen</strong>? </li> <li>Wat is het <strong>start en einduur</strong> voor dagkampen en is er vooraf en nadien <strong>opvang</strong> voorzien?</li> </ul>',
+          'Festiviteit': '<p>Geef een enthousiaste omschrijving van de kermis. </p> <ul> <li>Welke <strong>attracties</strong> zijn er? </li> <li>Zijn er<strong> (rand)activiteiten</strong> (quiz, concerten, foodtrucks, marktje ...)?</li> </ul>',
+          'Route': '<p>Geef een enthousiaste omschrijving van de route. </p> <ul> <li>Leg je de route af <strong>te voet, per fiets, per boot </strong>…? </li> <li>Welke mogelijke <strong>tussenstops</strong> zijn er? </li> <li>Wat is de <strong>afstand</strong> of <strong>duur</strong> van de route?</li> <li>Is de route <strong>begeleid</strong>, met een gids, brochure of wegwijzers?</li> </ul>',
+          'Party of fuif': '<p>Geef een enthousiaste omschrijving van de party of fuif. Welke <strong>dj’s</strong> of <strong>bands</strong> treden er op?</p>',
+          'Eet of drankfestijn': '<p>Geef een enthousiaste omschrijving van het eet -of drankfestijn. </p> <ul> <li>Wat staat er op het <strong>menu?</strong> </li> <li>Gaat de opbrengst naar een<strong> goed doel</strong>? </li> <li>Is er <strong>(rand)animatie</strong>?</li> </ul>',
+          'Spel of quiz': '<p>Geef een enthousiaste omschrijving van de quiz of het spel. </p> <ul> <li>Over welke <strong>onderwerpen</strong> gaat de quiz?</li> <li>Heeft het spel een <strong>thema</strong>?</li> <li>Valt er iets te <strong>winnen</strong>?</li> <li>Is er een <strong>maximaal</strong> aantal <strong>deelnemers</strong>?</li> </ul>',
+          'Sport en beweging': '<p>Geef een enthousiaste omschrijving van de sportactiviteit. </p> <ul> <li>Welke <strong>sport</strong>(en) kan je beoefenen? </li> <li>Welk <strong>materiaal</strong> wordt (niet) voorzien?</li> </ul>',
+          'Monument': '<p>Geef hier een wervende omschrijving van het monument. Geef ook aan indien het monument slechts beperkt opengesteld is (vb. enkel salons).</p>'
+        },
```

Comment:
Not a big fan of all this HTML inside the translations. When we rebuild this in Vue/React we should look for a better solution for cases like this. But for now, it's definitely good enough. Just something to keep in mind.